plugin/utils/gapic_utils.py

```diff
@@ -99,29 +99,19 @@ def read_from_gapic_yaml(request):
     else:
         gapic_yaml = {}
 
-    # It is possible we got a "GAPIC v2", or no GAPIC YAML at all.
-    # GAPIC v2 is a stripped down GAPIC config that moves most of the relevant
-    # configuration, including resource names, over to annotations.
-    # No GAPIC YAML at all means all config is in annotations.
-    #
-    # If either of these situations apply, "build back" what the GAPIC v1
-    # would have looked like, allowing the rest of the logic to stay the
-    # same.
-    #
-    # Prop 65 Warning: The GAPIC config is known to the state of California
-    # to cause cancer and reproductive harm. The reason we do not refactor
-    # away from it here is because this tool is supposed to have a short
-    # shelf life, and it is safer to be backwards-looking than
-    # forward-looking in this case.
+    # Resource name configurations between gapic yaml v1 and gapic yaml v2 +
+    # proto annotations have diverged. If we got a "GAPIC v2" or no GAPIC
+    # YAML at all, we build a gapic_config from both GAPIC YAML and
```

Comment:
does it mean that "we build `gapic_config_v2`"?

Reply:
This actually refers to an instance of the `GapicConfig` object in this file. Updated to make it more clear.